### PR TITLE
Feature/panels ref EPs

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
😺

Dual Merge: 
- (TBD)
- (TBD)

Branched off of https://github.com/MetaPhase-Consulting/mock-fsbid/pull/330, so this looks a bit inflated

### CODE REVIEW

File: src/services/lookups.js [hunk 1]
```diff
@@ -65,12 +67,22 @@ const getCommuterPosts = CommuterPosts => async () => {
...
     const data = await model.fetchAll()
-    const results = data.serialize().map(d => (
-      _.pick(d, pickProps)
-    ))
+    const results = data.serialize().map(d => {
+      let x = _.pick(d, pickProps);
+      if(mapObj) {
+        Object.keys(mapObj).forEach(k => {
+          if(x[k]){
+            x[mapObj[k]] = x[k];
```
Comment (elizabeth-jimenez) [inline src/services/lookups.js:78]: @scott062 here is the magic

**Review by scott062 — APPROVED** [review]

File: src/services/agendas.js [hunk 2]
```diff
@@ -365,7 +365,11 @@ const getPanelDates = async (filsCols, query) => {
   }
 }
 
-const getPanels = async (filsCols) => {
+const getPanels = async (filsCols, query) => {
+  let numOfResults = query['rp.pageRows'];
+
+  numOfResults = Number.isInteger(parseInt(numOfResults)) ? numOfResults * 7 : 200;
```
Comment (scott062) [inline src/services/agendas.js:371]: 👀  why?

Comment (elizabeth-jimenez) [reply src/services/agendas.js:371]: To TRY to get around the janky pagination

Comment (elizabeth-jimenez) [reply src/services/agendas.js:371]: check it out in swagger, it almost fully works lol

**Pull request merged**